python/hail/methods/qc.py

```diff
@@ -29,58 +30,43 @@ def sample_qc(dataset, name='sample_qc') -> MatrixTable:
     This method computes summary statistics per sample from a genetic matrix and stores the results as
     a new column-indexed field in the matrix, named based on the `name` parameter.
```

Comment:
I think this could be clearer that a new column-indexed *struct* field is added, with fields as defined below. My first reading it wasn't immediately clear that `dp_stats` and `gq_stats` are fields in `sample_qc` and not separate top-level fields.